### ISSUE TITLE
[Issuer] The great big ETO steps cleanup

### ISSUE THREAD

**Issue by andywalton** [Neufund/platform-frontend#3258]
**Background**
After running through the Issuer flow we noted a number of minor issues. The focus for this ticket is to cleanup the following things to improve the usability of the ETO experience.

1. Delete step numbers. Correctly word each steps Title and tell the Issuer what they need to do next 
2. Copy change and quick fixes through
3. Check the status colouring and wording
4. Remove widgets that aren't needed at specific steps in the process - ie. don't display the Whitelist widget when you're in Public sale - it's something you can't do anymore.

The above should follow Aga's spreadsheet (Slide 7) and notes in this slidedeck:
https://docs.google.com/presentation/d/1v2gQFfd_gCI1XDgoDPCVFQpjEpqXbzYg-ehOHILHUlQ/edit#slide=id.p

I'd also recommend that @Vanilizer and the frontend dev doing this work possibly have a hack day together as it will be much quicker and easier to make this layout and mostly content changes together.

```
TL;DR: 
For each step in the process - we should make sure we display only what we need to and make sure copy is checked and changed
```

**Designs**
Note required

**Acceptance criteria**
Aga is happy. :)

**Technical notes**
Non required

**Testing notes**
@Vanilizer tests & QAs this.
